tests/console/yast2_registration.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: check if an unregistered system can be registered and if
+#          enabling and disabling extensions correctly work.
+# Maintainer: Paolo Stivanin <pstivanin@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use utils 'zypper_call';
+
+sub register_system_and_add_extension {
+    wait_screen_change { type_string get_var "SCC_EMAIL" };
+    send_key "tab";
+    wait_screen_change { type_string get_var "SCC_REGCODE" };
+    send_key "alt-n";
+    assert_screen("yast2_registration-ext-mod-selection", timeout => 60);
+    # enable Web and Scripting Module
+    for my $i (0 .. 25) {
```

Comment:
are you aware about `send_key_until_needlematch` function ? it looks more suitable here , counting 25 keydown press in lagging openQA environment looks error  prone

Reply:
yes, I know that function. The problem is that between SLE 15 and SLE 12 there are differences, so I would need to create lots of needles (and maintaining them, of course). I thought that, maybe, in such a case using that workaround would be better. What do you think?
If lag could be an issue then I could add a `sleep 1`, for example

Reply:
The purpose of that code is simply to go 'to the bottom', that's why the `25`. I looked for a 'PgDn' button, but found nothing.

Reply:
just try 'pgdn'  - see https://github.com/os-autoinst/os-autoinst/blob/f59b6c01f5f7fe5b212182c2abee2ddd3443eaa4/consoles/VNC.pm#L563

Reply:
Already tried that, but didn't work :(

Reply:
@Soulofdestiny  do you know about any issues with pgdown ?

Reply:
@asmorodskyi it's not an issue with openqa, but how yast2-registration works. I tried on a local vm and pgdn didn't work there either.